### PR TITLE
Release Godot 4.1.1 stable and publish release notes

### PR DESCRIPTION
**Pull request by YuriSizov**

### CODE REVIEW

**Review by adamscott — APPROVED** [review]
Even if there's no :nail_care: nor :bomb: in the article, I'm approving this post.

File: collections/_article/maintenance-release-godot-4-1-1.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+excerpt: "Hot on the heels of its parent release, here's Godot 4.1.1 — the first maintenance release of the 4.1 branch, with more stability and workflow improvements."
+categories: ["release"]
+author: Yuri Sizov
+image: /storage/blog/covers/maintenance-release-godot-4-1-1.webp
+image_caption_title: Gunforged
+image_caption_description: A game by Firebelley
+date: 2023-07-17 17:00:00
+---
+
+The release of Godot 4.1 has been [just a couple of weeks ago](/article/godot-4-1-is-here), and we are ready for the first maintenance release, — addressing some of the bugs and annoyances that have slipped into the stable build.
```
Comment (clayjohn) [inline collections/_article/maintenance-release-godot-4-1-1.md]:
```suggestion
The release of Godot 4.1 was [just a couple of weeks ago](/article/godot-4-1-is-here) and we are ready for the first maintenance release — addressing some of the bugs and annoyances that have slipped into the stable build.
```

File: collections/_article/maintenance-release-godot-4-1-1.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+- Command-line export of C#/.NET projects should no longer drop random files ([GH-79173](https://github.com/godotengine/godot/pull/79173)). Your exports may have had arbitrary resources missing (not C# scripts, but images, for instance), if you were exporting your project with CLI. This should no longer happen.
+
+- Custom export options which you can define with an `EditorExportPlugin` are now correctly restored on the editor restart ([GH-79025](https://github.com/godotengine/godot/pull/79025)). Previously the usability of this freshly added feature was limited due to data loss between sessions.
+
+- For Linux users there is a potential fix for freezes when interacting with menus on Wayland ([GH-79143](https://github.com/godotengine/godot/pull/79143)). This had been a hard to identify and debug issue, but our local Wayland enthusiasts managed to pinpoint the likely cause and validate that the unwanted behavior was addressed.
```
Comment (clayjohn) [inline collections/_article/maintenance-release-godot-4-1-1.md]:
```suggestion
- For Linux users there is a potential fix for freezes when interacting with menus on Wayland ([GH-79143](https://github.com/godotengine/godot/pull/79143)). This was a hard to identify and debug issue, but our local Wayland enthusiasts managed to pinpoint the likely cause and validate that the unwanted behavior was addressed.
```

**Review by clayjohn — COMMENTED** [review]
Just some small grammar improvements, otherwise it looks great!

**Comment by YuriSizov** [comment]
Thanks for reviews!

**Pull request merged**